package.json

```diff
@@ -53,6 +50,7 @@
   "engines": {
     "node": ">=4"
   },
+  "files":["@hoodie/admin", "@hoodie/client", "@hoodie/server"],
```

Comment:
the `"files"` key in the `package.json` is not for names of dependencies, but for file names and paths that are part of the repository, please read the description at that’s not how you use `files`. See the documentation at https://docs.npmjs.com/files/package.json#files